plugins/woocommerce-admin/client/customize-store/design-with-ai/services.ts

```diff
@@ -50,17 +50,14 @@ export const parseLookAndToneCompletionResponse = (
 		}
 	} catch {
 		recordEvent(
-			'customize_your_store_look_and_tone_ai_completion_response_error',
+			'customize_your_store_ai_wizard_completion_response_error',
```

Comment:
I think this needs to be more specific because we'll have another ai completion api call for the site assembler, and that'll come with its own response parsing

Alternatively we can specify a property indicating the api call

Reply:
Good point! I reverted the change in e2a9f3949bb405545c44d9690951f1c2328da604 🙂